### DISCUSSION TITLE
Pruning database

### DISCUSSION THREAD

**Discussion by The-Elevator-Guy** [chuot/rdio-scanner#168] [General]
I’ve had this all working now for a couple weeks. The only problem I have, is that the app will randomly hang up. The log says “database pruning”. I do have the app set to prune every 2 days. 

Setup - Windows 10, Unitrunker-Virtual Audio Cable-Trunking Recorder-Rdio

**Comment by chuot** [comment]
There was a problem about this with previous versions. Make sure you are using the latest version, which is v6.2.5.

**Reply by The-Elevator-Guy** [reply]
Yes, it is version 6.2.5. Still doing it as of today

**Comment by chuot** [comment]
Can you try to boost the dirwatch delay from the default 2000ms to something like 5000ms to see if your issue still persist?
